pkg/registry/apis/secret/accesscontrol.go

```diff
@@ -0,0 +1,225 @@
+package secret
+
+import (
+	"context"
+
+	claims "github.com/grafana/authlib/types"
+	"github.com/grafana/grafana/pkg/apimachinery/identity"
+	"github.com/grafana/grafana/pkg/apimachinery/utils"
+	secretv0alpha1 "github.com/grafana/grafana/pkg/apis/secret/v0alpha1"
+	"github.com/grafana/grafana/pkg/services/accesscontrol"
+	"k8s.io/apiserver/pkg/authorization/authorizer"
+)
+
+const (
+	// SecureValues
+	ActionSecretsManagerSecureValuesWrite    = "secrets-manager.securevalues:write"    // CREATE + UPDATE.
+	ActionSecretsManagerSecureValuesDescribe = "secrets-manager.securevalues:describe" // GET.
```

Comment:
To be consistent with other RBAC permissions: 
```suggestion
	ActionSecretsManagerSecureValuesRead = "secrets-manager.securevalues:read" // GET.
```